params/config.go

```diff
@@ -567,7 +567,7 @@ type BorConfig struct {
 	BurntContract              map[string]string      `json:"burntContract"`              // governance contract where the token will be sent to and burnt in london fork
 	JaipurBlock                *big.Int               `json:"jaipurBlock"`                // Jaipur switch block (nil = no fork, 0 = already on jaipur)
 	DelhiBlock                 *big.Int               `json:"delhiBlock"`                 // Delhi switch block (nil = no fork, 0 = already on delhi)
-	ParallelUniverseBlock      *big.Int               `json:"parallelUniverseBlock"`      // TODO: update all occurrence, change name and finalize number (hardfork for block-stm related changes)
+	NapoliBlock                *big.Int               `json:"napoliBlock"`                // TODO: update all occurrence, change name and finalize number (hardfork for block-stm related changes)
```

Comment:
Minor one: this won't only be block-stm related HF, right?

Reply:
Yeah, sorry, my bad. Discussed with @temaniarpit27, this will be Cancun Block in bor. In Erigon it will be called Napoli.